FAQ.md

```diff
@@ -0,0 +1,22 @@
+# Foire Aux Questions
+
+Cette page répertorie les questions fréquemment rencontrées au sujet de l'environnement technique d'OpenFisca.  
+N'hésitez pas à l'[enrichir](https://github.com/openfisca/tutorial/edit/master/FAQ.md) de vos expériences ! :slightly_smiling_face: 
+
+
+> Sous Windows, il est supposé que vous disposez d'un terminal bash (i.e [GitBash](https://gitforwindows.org) ou assimilé).
+
+## GIT
```

Comment:
Why is `git` first and `openfisca python` second ?

Reply:
Reversed.